src/nccl_ofi_sendrecv.cpp

```diff
@@ -1337,6 +1334,17 @@ static int sendrecv_recv_comm_alloc_and_reg_flush_buff(struct fid_domain *domain
 	return ret;
 }
 
+
+static int sendrecv_fl_req_entry_init(void *entry)
+{
+	nccl_net_ofi_sendrecv_req_t *req =
+		static_cast<nccl_net_ofi_sendrecv_req_t *>(entry);
+	req->base.test = sendrecv_req_test;
+	req->state = NCCL_OFI_SENDRECV_REQ_CREATED;
+	return 0;
```

Comment:
NIC: an empty line between the body and the return is pretty standard in our code.